utils/launch.sh

```diff
@@ -73,8 +74,12 @@ done
 which netstat >/dev/null 2>&1 \
     || die "Must have netstat installed"
 
-netstat -ltn | grep -qs ":${PORT} .*LISTEN" \
-    && die "Port ${PORT} in use. Try --listen PORT"
+case $(uname -s) in
+   "FreeBSD") netstat -lan | grep -qs ".${PORT} .*LISTEN" && PORTUSED=true ;;
+   "Linux") netstat -ltn | grep -qs ":${PORT} .*LISTEN" && PORTUSED=true ;;
```

Comment:
`netstat -lan` seems to work on Linux as well, so you should be able to merge these instead.